gallery/src/data-visualization/ReactVisStackedGraphPage.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxCode, NxP, NxTable, NxTableBody, NxTableCell,
+  NxTableHead, NxTableRow } from '@sonatype/react-shared-components';
+import { GalleryDescriptionTile, GalleryExampleTile } from '../gallery-components/GalleryTiles';
+import ReactVisStackedGraphExample from './examples/ReactVisStackedGraphExample';
+
+const reactVisStackedGraphExampleCode = require('./examples/ReactVisStackedGraphExample?raw');
+
+const ReactVisStackedGraphPage = () =>
+  <>
+    <GalleryDescriptionTile >
+      <NxP>
+        React-vis also supports stacked graphs where two or more graphs can be stacked on top of each other.
+      </NxP>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Basic API Reference</h3>
+        </header>
+        <NxTable>
```

Comment:
This page doesn't actually introduce any components that aren't already documented elsewhere, so I think this props table is redundant